tests/suites/helpers.function

```diff
@@ -20,19 +25,6 @@
 #include <setjmp.h>
 #endif
 
-#ifdef _MSC_VER
-#include <basetsd.h>
-typedef UINT8 uint8_t;
-typedef INT32 int32_t;
-typedef UINT32 uint32_t;
-#define strncasecmp _strnicmp
-#define strcasecmp _stricmp
-#else
-#include <stdint.h>
-#endif
-
-#include <string.h>
-
 #if defined(__unix__) || (defined(__APPLE__) && defined(__MACH__))
 #include <unistd.h>
 #include <strings.h>
```

Comment:
Does this line `#include <strings.h>` forgotten to remove in 24a63f7?
```suggestion

```

Reply:
Indeed, I got this one wrong and somehow forgot to record that in my rebase log. (Or maybe rerere bit? Would be hard to find out by now.) I pushed a new history where I changed this commit.